gitium/inc/class-gitium-submenu-status.php

```diff
@@ -120,6 +127,31 @@ public function save_changes() {
 		$this->success_redirect( sprintf( __( 'Pushed commit: `%s`', 'gitium' ), $commitmsg ) );
 	}
 
+	public function pull_changes() {
+		$gitium_pull_changes = filter_input(INPUT_POST, 'GitiumSubmitPullChanges', FILTER_SANITIZE_FULL_SPECIAL_CHARS);
+		
+		$commits   = array();
+		$commitmsg = sprintf( 'Merged changes from %s on %s', $_SERVER['SERVER_NAME'], date( 'm.d.Y' ) );
+	    
+		if ( ! isset( $gitium_pull_changes ) ) {
+			return;
+		}
+
+		check_admin_referer( 'gitium-admin' );
+		
+		gitium_enable_maintenance_mode() or wp_die( __( 'Could not enable the maintenance mode!', 'gitium' ) );
+		
+		if ( $this->git->is_dirty() && $this->git->add() > 0 ) {
+			$commits[] = $this->git->commit( $commitmsg ) or trigger_error( 'Could not commit local changes!', E_USER_ERROR );
+		}
+		
+		gitium_disable_maintenance_mode();
+		
+		gitium_merge_and_push( $commits ) or trigger_error( 'Failed merge & push: ' . serialize( $this->git->get_last_error() ), E_USER_ERROR );
```

Comment:
This function does merge and push more than pull ... is there a function that does only pull ??

Reply:
Maybe better change the meaning of `Save changes` button as this pull is too similar to it.

Reply:
For example Change the name of  the button from`Save Changes` -> `Pull Changes` if there is `$behind_commits > 0` and let the same save_changes function run updated acordingly.